pkgs/data/themes/tokyo-night-gtk-packages/default.nix

```diff
@@ -0,0 +1,105 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, unstableGitUpdater
+, gtk-engine-murrine
+, gnome-themes-extra
+}:
+
+# Why do we do this? This package has a LOT of files! The user may not need
+# some of these themes and icons and the ~100k extra files in them.
+# Why not just use passthru for these? See: https://github.com/NixOS/nixpkgs/issues/211340
+let
+  icons = {
+    dark = base { tkIcon = "Dark"; tkTheme = ""; };
+    dark-cyan = base { tkIcon = "Dark-Cyan"; tkTheme = ""; };
+    light = base { tkIcon = "Light"; tkTheme = ""; };
+    moon = base { tkIcon = "Moon"; tkTheme = ""; };
+  };
+
+  themes = {
+    dark-b = base { tkIcon = ""; tkTheme = "Dark-B"; };
+    dark-bl = base { tkIcon = ""; tkTheme = "Dark-BL"; };
+    dark-b-lb = base { tkIcon = ""; tkTheme = "Dark-B-LB"; };
+    dark-bl-lb = base { tkIcon = ""; tkTheme = "Dark-BL-LB"; };
+    storm-b = base { tkIcon = ""; tkTheme = "Storm-B"; };
+    storm-bl = base { tkIcon = ""; tkTheme = "Storm-BL"; };
+    storm-b-lb = base { tkIcon = ""; tkTheme = "Storm-B-LB"; };
+    storm-bl-lb = base { tkIcon = ""; tkTheme = "Storm-BL-LB"; };
+  };
+
+  base = {
+    tkIcon ? ""
+  , tkTheme ? ""
+  }: stdenvNoCC.mkDerivation (finalAttrs: {
+    pname = "tokyo-night-gtk-packages" + (
```

Comment:
```suggestion
    pname = "tokyo-night-gtk" + (
```

Reply:
Ended up going for something a bit different.